src/Core/Users/classUser.php

```diff
@@ -0,0 +1,67 @@
+<?php
+namespace Meetup\Login\Core;
+use Meetup\src\Core\Users\UserInterface
+Class User  implements  UserInterface{
+private $firstName;  
+private $lastName;  
+private $createdAt;  
+private $updatedAt;  
+private $email;  
+private $password;  
+
+	public function __construct($email, $password){
+		$this->setEmail($email);
+		$this->setPassword($password);
+	}
+	public function setPassword($password){
+		$this->password = $password;
+	}
+	public function setEmail($email){
+		$this->email = $email;
+	}
+	public function checkPassword(){
+		$sql = "SELECT * FROM user WHERE email='".$this->email."' AND password='".md5($this->password)."' ";
+		$query = mysql_query($sql);
+		if(mysql_num_rows() > 0)
+			return mysql_fetch_row($query);
+		else
+			return false;
+	}
+	
+	public function setFirstName($firstName){
+		$this->firstName = $firstName;
+	}
+	public function getFirstName(){
+		return $this->firstName;
+	}
+	public function setLastName($lastName){
+		$this->lastName = $lastName;
+	}
+	public function getLastName(){
+		return $this->lastName;
+	}
+    public function getCreatedAt(){
+		return $this->createdAt;
+	}
+    public function setCreatedAt($createdTime){
+		$this->createdAt = $createdTime;
+	}
+    public function getUpdatedAt(){
+		return $this->updatedAt;
+	}
+    public function setUpdatedAt($datetime){
+		$this->updatedAt = $dateTime;
+		$sql = "SELECT id FROM user WHERE $email='".$this->email."' updated_at='".$datetime."' ";
```

Comment:
Why the query? You are not doing that in `setCreatedAt`but in `setUpdatedAt`. Can you please explain the reason?